legend-engine-xts-analytics/legend-engine-xts-analytics-mapping/legend-engine-xt-analytics-mapping-pure/src/main/resources/core_analytics_mapping/modelCoverage/utility.pure

```diff
@@ -60,28 +60,6 @@ function meta::analytics::mapping::modelCoverage::utility::getFunctionPath(f:Fun
    $f->elementToPath()->replace($f.name->toOne(), $f.functionName->toOne());
 }
 
-
-function {doc.doc = 'Get the parent or super types of the provided class'}
```

Comment:
you're sure nothing is using these functions?

Reply:
We are using this functions. Just moved it to different place. I moved it to class analytics utility